File: python/Vehicle.py
# platNomor 
# merk
# tahunProduksi
# warna

class Vehicle(): 
    __platNomor = ""
    __merk = ""
    __tahunProduksi = ""
    __warna = ""

    def __init__(self, platNomor="", merk="", tahunProduksi="", warna=""):
        self.__platNomor = platNomor
        self.__merk = merk
        self.__tahunProduksi = tahunProduksi
        self.__warna = warna

    # getset
        
    def getplatNomor(self):
        return self.__platNomor
    
    def getmerk(self):
        return self.__merk
    
    def gettahunProduksi(self):
        return self.__tahunProduksi
    
    def getwarna(self):
        return self.__warna
    
    
    
    def setplatNomor(self, platNomor):
        self.__platNomor = platNomor

    def setmerk(self, merk):
        self.__merk = merk

    def settahunProduksi(self, tahunProduksi):
        self.__tahunProduksi = tahunProduksi

    def setwarna(self, warna):
        self.__warna = warna

    

File: c++/Vehicle.cpp
#pragma once
#include <iostream>
#include <string>

using namespace std;

class Vehicle {
private:
    string platNomor;
    string merk;
    string tahunProduksi;
    string warna;

public:
    Vehicle(string platNomor="", string merk="", string tahunProduksi="", string warna="") {
        this->platNomor = platNomor;
        this->merk = merk;
        this->tahunProduksi = tahunProduksi;
        this->warna = warna;
    }

    // getter
    string getPlatNomor() {
        return platNomor;
    }

    string getMerk() {
        return merk;
    }

    string getTahunProduksi() {
        return tahunProduksi;
    }

    string getWarna() {
        return warna;
    }

    // setter
    void setPlatNomor(string platNomor) {
        this->platNomor = platNomor;
    }

    void setMerk(string merk) {
        this->merk = merk;
    }

    void setTahunProduksi(string tahunProduksi) {
        this->tahunProduksi = tahunProduksi;
    }

    void setWarna(string warna) {
        this->warna = warna;
    }
};
